buildconfig/pygame-stubs/sprite.pyi

```diff
@@ -27,41 +14,37 @@ class Sprite:
     image: Optional[Surface] = None
     rect: Optional[Rect] = None
     def __init__(self, *groups: AbstractGroup) -> None: ...
-    def update(self, *args, **kwargs) -> None: ...
+    def update(self, *args: str, **kwargs: int) -> None: ...  # TODO: verify
     def add(self, *groups: AbstractGroup) -> None: ...
     def remove(self, *groups: AbstractGroup) -> None: ...
     def kill(self) -> None: ...
     def alive(self) -> bool: ...
-    def groups(self) -> List[AbstractGroup]: ...
+    def groups(self) -> Sequence[AbstractGroup]: ...
 
 class DirtySprite(Sprite):
     dirty: int
     blendmode: int
     source_rect: Rect
     visible: int
-    _layer: int
-    def _set_visible(self, value: int) -> None: ...
-    def _get_visible(self) -> int: ...
 
 class AbstractGroup:
-    spritedict = Dict[Sprite, int]
-    lostsprites = List[int]  # I think
+    spritedict = Mapping[Sprite, int]
+    lostsprites = Sequence[int]  # I think
     def __init__(self) -> None: ...
     def __len__(self) -> int: ...
     def __iter__(self) -> Iterator[Sprite]: ...
     def copy(self) -> AbstractGroup: ...
-    def sprites(self) -> List[Sprite]: ...
+    def sprites(self) -> Sequence[Sprite]: ...
     def add(self, *sprites: Sprite) -> None: ...
     def remove(self, *sprites: Sprite) -> None: ...
     def has(self, *sprites: Sprite) -> bool: ...
-    def update(self, *args, **kwargs) -> None: ...
-    def draw(self, surface: Surface) -> None: ...
+    def update(self, *args: str, **kwargs: int) -> None: ...  # TODO verify
+    def draw(self, surface: Surface) -> Sequence[Rect]: ...
```

Comment:
I think this was correct before? The default Group implementation returns None.

Reply:
Fixed